src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,14 @@
+package org.example;
+
+import org.example.inspector.SongInspectorImpl;
+
+public class Main {
+  public static void main(String[] args) {
+    SongInspectorImpl songInspectorImpl = new SongInspectorImpl();
```

Comment:
Assign it to `SongInspector songInspector`, otherwise why having the interface at all?